File: Admission/app/src/main/java/com/example/adeeliftikhar/admission/Internet/CheckInternetConnectivity.java
package com.example.adeeliftikhar.admission.Internet;

import android.content.Context;
import android.net.ConnectivityManager;
import android.net.NetworkCapabilities;
import android.net.NetworkInfo;

public class CheckInternetConnectivity {

    public static boolean isConnected(Context context) {
//        Used to check Internet Connection...

        ConnectivityManager connectivityManager = (ConnectivityManager) context.getSystemService(Context.CONNECTIVITY_SERVICE);
        if (connectivityManager != null) {
            NetworkInfo[] networkInfo = connectivityManager.getAllNetworkInfo();
            if (networkInfo != null) {
                for (int i = 0; i < networkInfo.length; i++) {
                    if (networkInfo[i].getState() == NetworkInfo.State.CONNECTED) {
                        return true;
                    }
                }
            }
        }
        return false;
    }
}

File: Messenger/app/src/main/java/com/example/adeeliftikhar/messenger/Internet/CheckInternetConnectivity.kt
package com.example.adeeliftikhar.messenger.Internet

import android.content.Context
import android.net.ConnectivityManager
import android.net.NetworkCapabilities
import android.net.NetworkInfo

object CheckInternetConnectivity {

    fun isConnected(context: Context): Boolean {
        //        Used to check Internet Connection...

        val connectivityManager = context.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
        if (connectivityManager != null) {
            val networkInfo = connectivityManager.allNetworkInfo
            if (networkInfo != null) {
                for (i in networkInfo.indices) {
                    if (networkInfo[i].state == NetworkInfo.State.CONNECTED) {
                        return true
                    }
                }
            }
        }
        return false
    }
}